library/Barberry/Cache.php

```diff
@@ -1,50 +1,51 @@
 <?php
 namespace Barberry;
 
+use function Barberry\file\als;
+
 class Cache {
 
     private $path;
 
     public function __construct($path) {
-        $this->path = rtrim($path,'/') . '/';
+        $this->path = als($path);
+        set_error_handler(array($this, 'errorHandler'));
```

Comment:
It's not a good idea to use error handlers in libraries, we should honor global error handlers set by the top level application.